### PR TITLE
Add missing badges

### PR DESCRIPTION
**Pull request by dhoppe**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+run the integration tests against Centos 6.5 with.
+
+    BEAKER_set=centos-64-x64 bundle exec rake acceptances
+
+If you don't want to have to recreate the virtual machine every time you
+can use `BEAKER_DESTROY=no` and `BEAKER_PROVISION=no`. On the first run you will
+at least need `BEAKER_PROVISION` set to yes (the default). The Vagrantfile
+for the created virtual machines will be in `.vagrant/beaker_vagrant_fies`.
+
+# vim: syntax=markdown
```
Comment (bastelfreak) [inline CONTRIBUTING.md]: this has to be removed

**Comment by bastelfreak** [comment]
@dhoppe we have the CONTRIBUTIG.md in .github/, and this is managed via modulesync. I'm not sure if we should have it twice in the repo. Can we symlink it or manage both via modulesync?

**Comment by dhoppe** [comment]
Regarding to this [blog post](https://github.com/blog/2111-issue-and-pull-request-templates), you can put `CONTRIBUTING.md`, `ISSUE_TEMPLATE.md` and `PULL_REQUEST_TEMPLATE.md` into the `.github` directory. But I do not know how `CONTRIBUTING.md` is supposed to work, since this is no template and shown anywhere. 

Actually all of our modules have the `CONTRIBUTING.md` and `.github/CONTRIBUTING.md`. The first one does not get rolled out via `modulesync` and is different than the other one.

I think we should move the file out of `.github`, because the user should be able to see this file quickly.

**Comment by bastelfreak** [comment]
It is linked to you if you open up a PR. I will open an issue in modulesync_config so we can decide it later.

**Pull request merged**